paddle/fluid/framework/ir/graph_pattern_detector.cc

```diff
@@ -958,6 +958,24 @@ PDNode *patterns::OperatorActivation::operator()(
   return activation_out;
 }
 
+PDNode *patterns::Squeeze2Transpose2::operator()() {
+  auto *preceding_op_out = pattern->NewNode(preceding_op_out_repr())
```

Comment:
I think that it should be named as `squeeze2_op_in`, because `preceding_op` is misleading